src/sonic-host-services/scripts/determine-reboot-cause

```diff
@@ -111,6 +111,9 @@ def find_hardware_reboot_cause():
     except ImportError:
```

Comment:
>ImportError [](start = 11, length = 11)

I think you use `ImportError` to determine test environment. This is fragile.
You'd better manipulate libraries in the test code, and not change scripts under test.
There are samples in [https://github.com/Azure/sonic-snmpagent](https://github.com/Azure/sonic-snmpagent)

Reply:
I agree that `ImportError` handling is not the best way to determine test env. In this change, based on the discussion offline, we have decided to use mock function to avoid hitting this code at all.